app/src/main/java/com/example/tmdb/screens/Details.kt

```diff
@@ -0,0 +1,498 @@
+                                modifier = Modifier
+                                    .padding(15.dp)
+                                    .align(
+                                        Alignment.Center
+                                    )
+                                    .clickable { Router.navigateTo(Screen.MainScreen(Router.lastHomeTab)) }
+                            )
+                        }
+                    }
+
+                    Image(
+                        painter = painterResource(id = R.drawable.tmdb),
+                        contentDescription = "TMDB App Bar",
+                        alignment = Alignment.Center,
+                        modifier = Modifier
+                            .fillMaxHeight()
+                            .fillMaxWidth()
+                            .wrapContentWidth(align = Alignment.CenterHorizontally)
+                            .padding(8.dp)
+                            .scale(3F)
+                    )
+                }
+
+            }
+        },
+        bottomBar = {
+            val selectedIndex = remember { mutableStateOf(1) };
+            BottomNavigation(
+                elevation = 10.dp,
+                backgroundColor = Colors.White100
+            ) {
+
+                BottomNavigationItem(
+                    icon = {
+                        Icon(
+                            imageVector = Icons.Default.Home,
+                            "Home",
+                            //tint = Colors.Grey200.copy(alpha = 0.6f),
+                            modifier = Modifier
+                        )
+                    },
+                    label = { Text(text = "Home" /*colors = Colors.Grey200.copy(alpha = 0.6f)*/) },
+                    selected = (selectedIndex.value == 1),
+                    onClick = {
+                        Router.navigateTo(Screen.MainScreen(MainScreenTab.HomeTab))
+                        selectedIndex.value = 1
+                    }, selectedContentColor = Colors.Blue700,
+                    unselectedContentColor = Colors.Grey200.copy(alpha = 0.6f)
+                )
+
+                BottomNavigationItem(
+                    icon = {
+                        Icon(
+                            imageVector = Icons.Default.Favorite,
+                            "Favorites",
+                            //tint = Colors.Blue700
+                        )
+                    },
+                    label = { Text(text = "Favorite", color = Colors.Blue700) },
+                    selected = (selectedIndex.value == 0),
+                    onClick = {
+                        Router.navigateTo(Screen.MainScreen(MainScreenTab.FavoriteTab))
+                        selectedIndex.value = 0
+                    }, selectedContentColor = Colors.Blue700,
+                    unselectedContentColor = Colors.Grey200.copy(alpha = 0.6f)
+                )
+            }
+        }) {
+        LazyColumn(Modifier.fillMaxSize()) {
```

Comment:
There is no need to put whole screen in `LazyColumn` as its main use is to cache items in smart way. Rather use `Column` with `Modifier.scrollable` instead.
The whole content of this column is pretty large, please separate it into smaller functions for subsections inside it, e.g. `Header`, `Actors` etc. All those functions could be in this same file and private.